storm-core/test/jvm/org/apache/storm/scheduler/resource/TestUtilsForResourceAwareScheduler.java

```diff
@@ -285,4 +288,72 @@ public static TopologyDetails findTopologyInSetFromName(String topoName, Set<Top
         }
         return ret;
     }
+
+    public static Map<SupervisorDetails, Double> getSupervisorToMemoryUsage(Cluster cluster, Topologies topologies) {
```

Comment:
This method and getSupervisorToCpuUsage are almost exactly identical. Can you combine them to reduce duplicate code?